uberfire-server/src/main/java/org/uberfire/server/locale/GWTLocaleHeaderFilter.java

```diff
@@ -89,55 +92,4 @@ private Locale getLocale(final ServletRequest request) {
         }
         return locale;
     }
-
-    static class CharResponseWrapper extends HttpServletResponseWrapper {
-
-        protected CharArrayWriter charWriter = new CharArrayWriter();
-
-        protected ServletOutputStream outputStream = new ServletOutputStream() {
-            @Override
-            public boolean isReady() {
-                return true;
-            }
-
-            @Override
-            public void setWriteListener(WriteListener writeListener) {
-                // TODO how to treat the listener?
-            }
-
-            @Override
-            public void write(int b) throws IOException {
-                charWriter.write(b);
-            }
-        };
-
-        protected PrintWriter writer = new PrintWriter(charWriter);
-
-        public CharResponseWrapper(final HttpServletResponse response) {
-            super(response);
-        }
-
-        @Override
-        public ServletOutputStream getOutputStream() throws IOException {
-            return outputStream;
-        }
-
-        @Override
-        public PrintWriter getWriter() throws IOException {
-            return writer;
-        }
-
-        @Override
-        public void flushBuffer() throws IOException {
-            // Don't remove this override!
```

Comment:
I feel the pain behind this comment :-) /cc @porcelli @manstis

Reply:
yeah.. -1 to remove this.

Reply:
I just find these code same to org.jboss.errai.common.server.FilterCacheUtil.CharResponseWrapper.
I think, it can reuse it.